app/api/users.py

```
@@ -0,0 +1,200 @@
+async def update_user(
+    user_id: str,
+    user_update: UserCreate,
+    token: Annotated[str, Depends(oauth2_scheme)],
+    db: AsyncSession = Depends(get_db)
+) -> UserResponse:
+    """Update user information (requires authentication)."""
+    from sqlalchemy import select
+    from uuid import UUID
+    
+    # Verify current user
+    current_user = await auth_service.get_current_user_from_token(db, token)
```

Ruff (0.12.2)

Local variable `current_user` is assigned to but never used

Remove assignment to unused variable `current_user`

(F841)